gsa/src/web/pages/results/row.js

```diff
@@ -83,14 +89,17 @@ const Row = ({
       <TableData flex align="center">
         {entity.qod.value} %
       </TableData>
-      <TableData flex align="center">
+      <TableData colSpan="2">
         <DetailsLink
           type="host"
           id={host.id}
           textOnly={!links}
         >
           {host.name}
         </DetailsLink>
+        {host.hostname &&
```

Comment:
You have defined `host.hostname` in models/results.js to be always a string. So this check should be `host.hostname.length > 0`